go.mod

```diff
@@ -30,7 +29,10 @@ require (
 	golang.org/x/time v0.0.0-20210611083556-38a9dc6acbc6
 )
 
-require go.elastic.co/ecszerolog v0.1.0
+require (
```

Comment:
Why do we multiple require blocks in the go.mod file and why did it change as part of this PR?

Reply:
accidental, i'll revert this change.